src/utils/endpoints/organizations-endpoints.js

```diff
@@ -0,0 +1,25 @@
+import getData from "../../utils/data/organizations-data";
+import { GET_ONE, GET_LIST, GET_MANY, GET_MANY_REFERENCE } from "react-admin";
+
+const apiUrl = "https://comments-microservice.herokuapp.com/v1";
+
+export default (type, params) => {
+  switch (type) {
+    case GET_MANY:
+    case GET_MANY_REFERENCE:
+    case GET_LIST:
+      return {
+        url: `${apiUrl}/msadmins/organizations`,
+        getData: getData,
+      };
+    case GET_ONE:
+      return {
+        url: `${apiUrl}/msadmins/organizations/${params.id}`,
+        getData: getData,
+      };
```

Comment:
missing `DELETE /msadmins/organizations/:organizationId` implementation

Reply:
Okay, will add it now.